package.json

```diff
@@ -14,27 +14,27 @@
     "test": "pnpm -F react test"
   },
   "devDependencies": {
-    "@changesets/cli": "^2.27.7",
+    "@changesets/cli": "^2.27.8",
     "@optiaxiom/shared": "workspace:^",
     "@rollup/pluginutils": "^5.1.0",
-    "@testing-library/jest-dom": "^6.4.8",
+    "@testing-library/jest-dom": "^6.5.0",
     "@testing-library/react": "^14.3.1",
     "@testing-library/user-event": "^14.5.2",
     "@types/react": "^18.3.3",
```

Comment:
explicitly holding back `@types/react` update to `18.3.5` as it conflicts with type generation for `Transition` component props fro some reason (guessing it has to do with the `data-size` prop)